### DISCUSSION TITLE
how to update my chezmoi repo

### DISCUSSION THREAD

**Discussion by jojojojojoj5564656465465** [twpayne/chezmoi#2206] [Q&A]
CHEZMOI CD
 GIT ADD .
GIT COMMIT -U "FOO"

git PUSH

But it doesn't work, How do you update your repo, Documentation should be more explicite how update repo

**Comment by twpayne** [comment]
There is not enough information in your message to help you. You say "it doesn't work" but you do not say what you expect to happen or what actually happens.

Maybe one of the [many articles on chezmoi](https://www.chezmoi.io/links/articles-podcasts-and-videos/) will help you?

**Comment by jojojojojoj5564656465465** [comment]
i would like to update chezmoi github repo from my local chezmoi repo

Also how can i do it with ssh and not token i saw on you site [SSH](https://www.chezmoi.io/user-guide/manage-different-types-of-file/#populate-sshauthorized_keys-with-your-public-ssh-keys-from-github)

I don't get how to do it because i have already on my chezmoi folder a private_dot_ssh

![Screenshot_20220711_184049](https://user-images.githubusercontent.com/45184918/178314781-6faffd2a-c1d8-41d2-bf81-1e8b05ddeba4.png)

**Comment by twpayne** [comment]
> i would like to update chezmoi github repo from my local chezmoi repo

Use `git push`.

From your question, I would strongly recommend reading some tutorials on git and working through chezmoi's quick start. Sorry, I cannot help you further.

**Comment by jojojojojoj5564656465465** [comment]
![image](https://user-images.githubusercontent.com/45184918/178618174-6219aef1-44c1-473b-a24b-f403884aabc0.png)

**Comment by jojojojojoj5564656465465** [comment]
I found the answer, i can edit .local then apply + autopush

I would rather sync from .config => .local but thank for your works
